### DISCUSSION TITLE
part-ii/2020-89/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#113] [Question comments]
Part II 2020 Logic and Set Theory: Paper 4, Section II, 16H 

https://questions.tripos.org/part-ii/2020-89/

**Comment by ArthurConmy** [comment]
All but iii) is bookwork / direct checks. Here's a solution without Zorn's lemma (directly).

Axiomatise ultrafilters by making a language with symbols (arity 0) $x_S$ for all $S \subseteq 2^\mathbb{ N}$ and also symbols $0$ and $1$ representing "not in filter" and "in ultrafilter" respectively and finally an arity 1 function $f$ that maps to $a_0$ or $a_1$ (by adding axioms) to represent the choice to include in the ultrafilter or not (I can comment the full list of axioms if wanted ...).

Then consider the axioms $f(x_{\{  1 \}}) = 0, f(x_{\{  2 \}}) = 0, ...$ . 

Finite subsets of the axiom set have models since we will only have finitely many $f(x_{\{  n \}}) = 0$ axioms so can choose some large $N$ for $\mathcal{U}_N$. So by compactness there's a model of all the axioms, which means an ultrafilter with no single element sets in it, which is clearly not any of the $\mathcal{U}_n$.

**Comment by artie2000** [comment]
Idea for the (I assume) intended solution to (iii):
Consider the set of filters, partially ordered by inclusion. What are the maximal elements?

**Reply by ArthurConmy** [reply]
Isn't showing maximal elements are ultrafilters non-trivial? i.e that when we add in some $A \subseteq \mathbb{ N}$ and create a new filter, we don't break any of the three conditions? [this was why I didn't finish this approach]

**Reply by artie2000** [reply]
Yes, there's some work to do. The "only sensible thing to do" works, though: extend a filter $\mathcal{M}$ to $\mathcal{M}'$ by intersecting your new point $A$ with everything in the filter, then adjoin all supersets of elements in $\mathcal{M}'$ to get a strictly bigger filter $\mathcal{M}''$.

I had some messy casework in my answer but in hindsight it can be done fairly neatly.
